pkgs/servers/mail/postfix/default.nix

```diff
@@ -104,7 +103,22 @@ in stdenv.mkDerivation rec {
       --prefix PATH ":" ${lib.makeBinPath [ coreutils findutils gnugrep gawk gnused ]}
   '';
 
-  passthru.tests = { inherit (nixosTests) postfix postfix-raise-smtpd-tls-security-level; };
+  passthru = {
+    tests = { inherit (nixosTests) postfix postfix-raise-smtpd-tls-security-level; };
+
+    updateScript = writeScript "update-postfix" ''
```

Comment:
Leaving the update-script as a separate file is more readable IMO.

Reply:
Moved to a separate file.